panacea/web/static/js/app.js

```diff
@@ -16,30 +16,28 @@ async function submitFile() {
 
     if (response.ok) {
       renderFileResponse(await response.json());
-      this.reset();
     } else {
-      // TODO: When API stops sending 200s change this to do the 'error' bit below
-      console.log('Request failed');
+      renderFileError(await response.json());
     }
+    this.reset();
   } catch (err) {
     console.log(err);
   }
 }
 
+const successResultMessage = document.getElementById('success-result-message');
+const errorResultMessage = document.getElementById('error-result-message');
+const successPanel = document.getElementById('success-panel');
+const errorPanel = document.getElementById('error-panel');
+
 const renderFileResponse = data => {
-  const successResultMessage = document.getElementById('success-result-message');
-  const errorResultMessage = document.getElementById('error-result-message');
-  const successPanel = document.getElementById('success-panel');
-  const errorPanel = document.getElementById('error-panel');
+    successResultMessage.innerHTML = JSON.stringify(data.drugs);
+    errorPanel.style.display = 'none';
+    successPanel.style.display = 'block';
+};
 
-  if (data.status === 'error') {
-      // TODO: When API stops sending 200s change this
-    errorResultMessage.innerHTML = data.message
+const renderFileError = data => {
+    errorResultMessage.innerHTML = data.message;
```

Comment:
Make `errorResultMessage` and `successResultMessage` local to just the functions that need them?